app/main.py

```diff
@@ -2,114 +2,325 @@
+                responseError["message"] = "Cannot filter with both arguments `language` and `type`"
+                return response.json(responseError, status=400)
+            if "type" in args:
+                languages = languages.filter(type=args["type"])
+            if "page" in args and int(args["page"]) > 1:
+                languages = languages.skip(int(args["page"])*ITEMS_PER_PAGE)
+                responseListObjects["page"] = args["page"]
 
-    languages = []
-    for d in my_db.languages.find(queryfilter):
-        d['_id'] = str(d['_id'])
-        languages.append(d)
-    return response.json({"message": "successfull", "data": languages})
+        responseListObjects["total_results"] = languages.count()
+        responseListObjects["total_pages"] = ceil(
+            languages.count() / ITEMS_PER_PAGE)
+        languages.limit(ITEMS_PER_PAGE)
+        responseListObjects["results"] = [d.serialize() for d in languages]
+        return response.json(responseListObjects)
+    except Exception as err:
+        responseError["message"] = err
+        return response.json(responseError, status=400)
 
 
 @api.route("/sentences")
-def get_sentences(request):
+def getSentences(request):
     """
-    return all sentences. If a parameter is given, then all sentences from that language
-    are returned
-    Endpoint example: /sentences?lang=fra
-    Query Parameter: (optional) lang 
+    Return all Sentences objects
+    Parameters in Request body
+    {
+        "page": (int)[Optional, default is 1]
+    }
     """
-    queryfilter = {}
-    if len(request.args) != 0:
-        if "lang" not in request.args:
-            return response.json({"message": "invalid query parameters "})
+    try:
+        sentences = Sentences.objects()
+        if request.json:
+            if "language" in request.json:
+                language = Languages.objects(
+                    language=request.json["language"]).first()
+                sentences = sentences.filter(lang=language)
+            if "page" in request.json and int(request.json["page"]) > 1:
+                sentences = sentences.skip(
+                    int(request.json["page"])*ITEMS_PER_PAGE)
+                responseListObjects["page"] = request.json["page"]
+
+        responseListObjects["total_results"] = sentences.count()
+        responseListObjects["total_pages"] = ceil(
+            sentences.count() / ITEMS_PER_PAGE)
+        sentences.limit(ITEMS_PER_PAGE)
+        responseListObjects["results"] = [d.serialize() for d in sentences]
+        return response.json(responseListObjects)
+    except Exception as err:
+        responseError['message'] = str(err)
+        return response.json(responseError, status=400)
+
+
+@api.route("/sentences", methods=["POST"])
+def saveSentences(request):
+
+    try:
+        if request.json:
+            postdata = request.json
+            if "text" not in postdata or "language" not in postdata:
```

Comment:
magic strings like these are almost always not good.

Reply:
As discussed, I will refactored this by using arrays like I did on line 222 and 223